src/worker/core/draft/genPlayers.ts

```diff
@@ -157,6 +157,669 @@ const genPlayers = async (
 					tids: [g.get("userTid")],
 				});
 			}
+		};
+			if (Math.random() < 0.75) {
```

Comment:
A lot of this code winds up being repetitive. Would probably be good to put it in a function or loop, like:

```
const easterEggInfos = [{
  firstName: "Augustus",
  hgt: 69,
  // .. all other parameters that vary here
}];
for (const info of easterEggInfos) {
  // all the code that's the same goes here
}
```

If you're not sure how to do that, no worries :)